notifique/src/main/java/com/nathanrassi/notifique/NotifiqueActivity.kt

```diff
@@ -29,30 +30,19 @@ class NotifiqueActivity : AppCompatActivity() {
     checkNotificationPermission()
   }
 
-  override fun onActivityResult(
-    requestCode: Int,
-    resultCode: Int,
-    data: Intent?
-  ) {
-    super.onActivityResult(requestCode, resultCode, data)
-    if (requestCode == notificationPermissionRequestCode) {
-      checkNotificationPermission()
-    }
-  }
-
   private fun checkNotificationPermission() {
     if (!NotificationManagerCompat.getEnabledListenerPackages(this).contains(packageName)) {
       val notificationSnackbar = Snackbar.make(
-          findViewById(android.R.id.content),
-          R.string.snackbar,
-          Snackbar.LENGTH_INDEFINITE
+              findViewById(android.R.id.content),
+              R.string.snackbar,
+              Snackbar.LENGTH_INDEFINITE
```

Comment:
revert spacing change?

Reply:
i was copying old code and didn't notice. fixed in next pr